Gujarati/પ્રાર્થના/પ્રાર્થના.md

```diff
@@ -0,0 +1,3 @@
+પ્રાર્થના એ પ્રભુ સાથે સંવાદ કર​વા નો સૌથી સરળ રસ્તો છે. હુ અંકિત આરદેશણા અહીં મારી એક પ્રિય પ્રાર્થના ટાંકુ છું.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.